### DISCUSSION TITLE
`mpv_config-2024.03.04`版本中，默认配置下，调节播放速度时音频会有一瞬间的卡顿。

### DISCUSSION THREAD

**Discussion by wathper** [dyphire/mpv-config#60] [General]
尝试将禁用下面的预设没有效果。
```
 [video-sync]
 profile-desc=当音频调整速度超过阈值时改变音视频同步模式，以避免明显的音高变化
 profile-cond=(speed * audio_speed_correction) > 0 and ((speed * audio_speed_correction) < 0.96 or (speed * audio_speed_correction) > 1.04) and not (p["estimated-vf-fps"] > 47 or p["display-fps"] > 120)
 profile-restore=copy-equal
 video-sync=display-tempo
 ```
 
- 启用 `af=rubberband`卡顿感减轻
 
- 启用
 `af=scaletempo=scale=1.0:stride=60:overlap=.20:search=14:speed=tempo`
或者
 `#af=scaletempo2=min-speed=0.25:max-speed=4.0:search-interval=30:window-size=20`
 之后再调节播放速度无卡顿
 
- 第三条参数官方手册变更为
`af=scaletempo2=min-speed=0.25:max-speed=8.0:search-interval=40:window-size=12`

变更说明：https://github.com/mpv-player/mpv/pull/12580

照他的说法新参数效果更好，但我1.5倍速时，听起来反而是旧的参数听感更佳。

**Comment by wathper** [comment]
还有一点
```
#af=scaletempo2=min-speed=0.25:max-speed=4.0:search-interval=30:window-size=20  
                                                                                # 此项参数为变速播放时默认自动激活的音频滤镜。在 min/max-speed=<N> 此速度范围外直接静音处理。scaletempo2 具有更高的音频质量
```
看描述，这个 af 是当前版本的默认值吗？但是注释掉这条参数后，变更播放速度又有卡顿。
尝试了注释掉这个 af 的同时，用官方的 mpv.exe 替换 掉`mpv_config-2024.03.04`中的，但是问题依旧。

**Comment by dyphire** [comment]
mpv 自身的问题建议打开 mpv 官方的 issue 反馈，相关 issue：https://github.com/mpv-player/mpv/issues/13694

**Reply by wathper** [reply]
原来如此。
